Simperium/src/main/java/com/simperium/android/AndroidClient.java

```diff
@@ -71,7 +70,7 @@ public AndroidClient(Context context){
             preferences.edit().putString(SESSION_ID_PREFERENCE, sessionToken).commit();
         }
 
-        mSessionId = String.format("%s-%s", Version.LIBRARY_NAME, sessionToken);
+        mSessionId = String.format("%s-%s", "android", sessionToken);
```

Comment:
This value was always `android` so removing the generated `Version` file doesn't have any impact on this.